cloudify_marketplace/provision/check_prerequisites

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
```

Comment:
Is it guaranteed some other way to always be run with bash? It will break if run with sh by mistake